src/management/PromptsManager.js

```diff
@@ -51,6 +51,23 @@ var PromptsManager = function(options) {
     options.tokenProvider
   );
   this.resource = new RetryRestClient(auth0RestClient, options.retry);
+
+  /**
+   * Retrieve custom text for a specific prompt and language.
+   * {@link https://auth0.com/docs/api/management/v2#!/Prompts/get_custom_text_by_language Custom Text endpoint}
+   *
+   *
+   * @type {external:RestClient}
+   */
+  var customTextByLanguageAuth0RestClient = new Auth0RestClient(
+    options.baseUrl + '/prompts/:prompt/custom-text/:language',
```

Comment:
This colon param syntax is specific to the rest-facade package and increases the coupling, making it harder to migrate to a different package if we need to. 

I think we should do as we do in [other managers](https://github.com/auth0/node-auth0/blob/master/src/auth/UsersManager.js#L124-L151) classes and create this URL here instead. Maybe checking for params presence as well, otherwise, the constructed URL might be invalid.

Reply:
Using this colon param syntax is already being used throughout the library https://github.com/auth0/node-auth0/blob/master/src/management/UsersManager.js#L56-L68 and other places.

Reply:
You are right 👍